website/src/components/genspectrum/GsMutationsOverTime.tsx

```diff
@@ -36,7 +45,11 @@ export const GsMutationsOverTime: FC<GsMutationsOverTimeProps> = ({
                 granularity={granularity}
                 lapisDateField={lapisDateField}
                 displayMutations={displayMutations ? JSON.stringify(displayMutations) : undefined}
+                useNewEndpoint={useNewEndpoint ?? false}
                 pageSizes={JSON.stringify(pageSizes ?? [10, 20, 30, 40, 50])}
+                initialMeanProportionInterval={JSON.stringify(
+                    initialMeanProportionInterval ?? { min: 0.05, max: 0.95 },
```

Comment:
Are we intentionally overwriting the default value here? I think it would be better to leave the default value to the component (i.e. not set the attribute if nothing is set in this wrapper here)

Reply:
maybe we could do this?

```
initialMeanProportionInterval={
  initialMeanProportionInterval
    ? JSON.stringify(initialMeanProportionInterval)
    : undefined
}
```

But why is it not done like this for pageSizes?

Reply:
> maybe we could do this?
> 
> ```
> initialMeanProportionInterval={
>   initialMeanProportionInterval
>     ? JSON.stringify(initialMeanProportionInterval)
>     : undefined
> }
> ```

Looks good to me :+1: 

> But why is it not done like this for pageSizes?

I'm not sure. The default here seems to be the one that the compents already have anyway.

Reply:
> But you're saying that if I just set it to undefined, it will be omitted?

It might be that it changes with React 19 (since it claims to have better web components support), but for that's how it works.